payment/src/main/java/com/kyriba/payment/web/controller/PaymentController.java

```diff
@@ -0,0 +1,103 @@
+package com.kyriba.payment.web.controller;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.kyriba.payment.domain.PaymentMethodType;
+import com.kyriba.payment.domain.PaymentStatus;
+import com.kyriba.payment.domain.dto.PaymentDto;
+import com.kyriba.payment.domain.vo.Amount;
```

Comment:
```java
import com.kyriba.payment.domain.PaymentMethodType;
import com.kyriba.payment.domain.PaymentStatus;
import com.kyriba.payment.domain.dto.PaymentDto;
import com.kyriba.payment.domain.vo.Amount;
```
is it OK to expose domain objects in API?

Reply:
Right now it is Ok for me.